### ISSUE TITLE
vmui: vmui doesn't respect -http.pathPrefix

### ISSUE THREAD

**Issue by laixintao** [VictoriaMetrics/VictoriaMetrics#2918]
**Describe the bug**
Then VictoriaMetrics start with `-http.pathPrefix`, `/vmui` still serves path at `/vmui` without prefix.

**To Reproduce**
Start process:

```
$ ps -ef | grep vic
root     152139      1 99 17:38 ?        03:16:18 /opt/victoriametrics/victoria-metrics-prod -promscrape.config /etc/victoriametrics/prom-conf.yaml -memory.allowedPercent 90 -http.pathPrefix /vm-sgw -storageDataPath /data/victoria-metrics/metrics-data
```

while other pathes work fine:

```
$ curl -I 127.0.0.1:8428/vm-sgw/metrics 
HTTP/1.1 200 OK
Content-Type: text/plain; charset=utf-8
X-Server-Hostname: sg-shopee-tpsre-d1pool-live-10-70-166-16
Date: Thu, 28 Jul 2022 10:06:09 GMT

$ curl -I 127.0.0.1:8428/vm-sgw/targets 
HTTP/1.1 200 OK
Content-Type: text/plain; charset=utf-8
X-Server-Hostname: sg-shopee-tpsre-d1pool-live-10-70-166-16
Date: Thu, 28 Jul 2022 10:06:12 GMT
```

vmui direct me to root path:

```
$ curl -I 127.0.0.1:8428/vm-sgw/vmui   
HTTP/1.1 301 Moved Permanently
Content-Type: text/html; charset=utf-8
Location: /vmui/?
X-Server-Hostname: sg-shopee-tpsre-d1pool-live-10-70-166-16
Date: Thu, 28 Jul 2022 10:06:17 GMT
```

though, if you visit `<prefix>/vmui/?`, it would be ok

```
$ curl -I 127.0.0.1:8428/vm-sgw/vmui/?
HTTP/1.1 200 OK
Accept-Ranges: bytes
Content-Length: 903
Content-Type: text/html; charset=utf-8
X-Server-Hostname: sg-shopee-tpsre-d1pool-live-10-70-166-16
Date: Thu, 28 Jul 2022 10:08:37 GMT

```

**Expected behavior**

don't redirect me to root

**Version**
The line returned when passing `--version` command line flag to the binary. For example:
```
$ /opt/victoriametrics/victoria-metrics-prod --version
victoria-metrics-20220714-170859-tags-v1.79.0-0-gd3116d986
```

**Used command-line flags**
Please provide the command-line flags used for running VictoriaMetrics and its components.

**Comment by valyala** [comment]
This issue should be fixed in the commit https://github.com/VictoriaMetrics/VictoriaMetrics/commit/6b0550c023c6acbf1589c20cbb0666666715c941 . This commit will be included in the next release. In the mean time it is possible to verify the bugfix by building VictoriaMetrics from this commit according to [these docs](https://docs.victoriametrics.com/#how-to-build-from-sources).

**Comment by valyala** [comment]
The bug should be fixed in [VictoriaMetrics v1.79.2](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing the issue then.

@laixintao , feel free re-opening the issue if it isn't fixed yet in VictoriaMetrics v1.79.2.
